Sources/scipio/PrepareCommands.swift

```diff
@@ -31,18 +44,36 @@ extension Scipio {
             let logLevel: Logger.Level = globalOptions.verbose ? .trace : .info
             LoggingSystem.bootstrap(logLevel: logLevel)
 
-            let runnerCachePolicies: [Runner.Options.CachePolicy]
-            switch cachePolicy {
+            // Show deprecation warning if cachePolicy is used
+            if cachePolicy != nil {
+                logger.warning("--cache-policy is deprecated. Please use --framework-cache-policy instead.")
```

Comment:
You must either forcefully override `--framework-cache-policy` or prevent it from running with an error, otherwise unintended behavior may occur.

We might not need to provide backward compatibility mode anymore 🤔
(Simply removing `-cache-policy` causes swift-argument-parser to throw an error)